Taxi/Taxi/Presenter/AddTaxiParty/AddTaxiParty.swift

```diff
@@ -42,7 +42,252 @@ struct AddTaxiParty: View {
+            RoundedButton("택시팟 생성", !checkAllInfoSelected()) {
+                // TODO: 택시팟 생성 유즈케이스와 연결하기!
+            }
+        }
+        .buttonStyle(.plain)
+    }
+
+    private var guideText: some View {
+        HStack {
+            Spacer()
+            Text("해당 사항이 맞으시면 택시팟 생성을 눌러주세요")
+                .caption()
+                .padding(.bottom, 12)
+            Spacer()
+        }
+    }
+
+    private var closeButton: some View {
+        Button {
+            dismiss()
+        } label: {
+            Image(systemName: "xmark")
+                .imageScale(.large)
+                .padding()
+        }
+    }
+}
+// MARK: - 여기서부턴 목적지를 설정하는 뷰입니다.
+extension AddTaxiParty {
+    private var destinationSelect: some View {
+        HStack(spacing: 16) {
+            ForEach(Place.destinations(), id: \.self) { destination in
+                destinationItem(destination)
+            }
+        }
+        .padding(EdgeInsets(top: 24, leading: 16, bottom: 24, trailing: 16))
+        .background(Color.addBackground)
+        .toInfoContainer(title: "어디로 가시나요?", selectedInfo: destination?.rawValue ?? "", toggle: step == .destination) {
+            changeStep(to: .destination)
+        }
+    }
+
+    private func destinationItem(_ place: Place) -> some View {
+        Button {
+            if destination != place {
+                destination = place
+                departure = nil
+                toNextStep()
+            }
+        } label: {
+            Text(place.rawValue)
+                .info()
+                .padding()
+                .frame(maxWidth: .infinity)
+                .roundedBackground(destination == place)
+        }
+    }
+}
+// MARK: - 날짜 선택 뷰
+extension AddTaxiParty {
+    private var datePicker: some View {
+        Button {
+            startDate = "1월 1일"
+            toNextStep()
+        } label: {
+            Text("추후 구현 예정")
+                .padding()
+        }
+        .toInfoContainer(title: "날짜를 선택해주세요", selectedInfo: startDate ?? "", toggle: step == .date) {
+            changeStep(to: .date)
+        }
+    }
+}
+// MARK: - 시간 선택 뷰
+extension AddTaxiParty {
+    private var timePicker: some View {
+        VStack(spacing: 0) {
+            hourSelector
+            Divider().background(Color.customGray.opacity(0.3))
+            minuteSelector
+        }
+        .background(Color.addBackground)
+        .toInfoContainer(title: "몇시에 모이고 싶으신가요?", selectedInfo: startTimeToString(), toggle: step == .time) {
+            changeStep(to: .time)
+        }
+    }
+
+    private func startTimeToString() -> String {
+        if let startHour = startHour, let startMinute = startMinute {
+            return "\(startHour)시 \(String(format: "%02d", startMinute))분"
+        } else {
+            return ""
+        }
+    }
+    private var hourSelector: some View {
+        ScrollView(.horizontal, showsIndicators: true) {
+            HStack(spacing: 17) {
+                ForEach(hourRange) { index in
```

Comment:
여기서 'Non-constant range: not an integer range' 경고가 뜨는데 id를 추가해 줘야 하지 않나요?

```suggestion
                ForEach(hourRange, id: \.self) { index in
```

Reply:
c2a62b6edccfecab4ad07edbea0996a2fa993478 에서 ㅂㅇ